ui/app/src/state/actions/dialogs.ts

```diff
@@ -200,10 +210,23 @@ export const pathSelectionDialogClosed = createAction('PATH_SELECTION_CLOSED');
 
 // region items notification
 
-export const showDeleteItemSuccessNotification = createAction('SHOW_DELETE_ITEM_SUCCESS_NOTIFICATION');
-export const showPublishItemSuccessNotification = createAction('SHOW_PUBLISH_ITEM_SUCCESS_NOTIFICATION');
+export const showDeleteItemSuccessNotification = createAction(
+  'SHOW_DELETE_ITEM_SUCCESS_NOTIFICATION'
+);
+export const showPublishItemSuccessNotification = createAction(
+  'SHOW_PUBLISH_ITEM_SUCCESS_NOTIFICATION'
+);
 export const showEditItemSuccessNotification = createAction('SHOW_EDIT_ITEM_SUCCESS_NOTIFICATION');
 export const showCopyItemSuccessNotification = createAction('SHOW_COPY_ITEM_SUCCESS_NOTIFICATION');
-export const showRevertItemSuccessNotification = createAction('SHOW_REVERT_ITEM_SUCCESS_NOTIFICATION');
+export const showCutItemSuccessNotification = createAction('SHOW_CUT_ITEM_SUCCESS_NOTIFICATION');
+export const showPasteItemSuccessNotification = createAction(
+  'SHOW_PASTE_ITEM_SUCCESS_NOTIFICATION'
+);
+export const showDuplicatedItemSuccessNotification = createAction(
+  'SHOW_DUPLICATED_ITEM_SUCCESS_NOTIFICATION'
+);
+export const showRevertItemSuccessNotification = createAction(
+  'SHOW_REVERT_ITEM_SUCCESS_NOTIFICATION'
+);
```

Comment:
Let's group all these system stuff under the new systemEvents actions file, only that, rename actions/systemEvents.ts to actions/system.ts